dir.props

```diff
@@ -485,6 +485,11 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
+  
+  <!-- Default Test platform to deploy the netstandard compiled tests to -->
+  <PropertyGroup>
+    <TestTFM Condition="'$(TestTFM)'==''>netcoreapp1.0</TestTFM>
```

Comment:
when doing build.cmd/build.sh from the root, does this will trigger running all tests against netcoreapp1.0? and if so, how this will happen? I am asking because there is a filtering part that I should be doing after you merge your changes.